openmc/mgxs_library.py

```diff
@@ -160,16 +96,16 @@ class XSdata(object):
         azimuthal angles times the number of polar angles, with the
         inner-dimension being groups, intermediate-dimension being azimuthal
         angles and outer-dimension being the polar angles.
-    scatter : numpy.ndarray
+    scatter_matrix : numpy.ndarray
```

Comment:
The comments here are getting quite long and repetitive. How do you feel about shortening the comments to just say what it is and then having a Note at the bottom that describes how the cross section arrays are structured? I'm working on adding the MGXS for delayed parameters that are not currently in the multi-group solver (e.g. chi_prompt, chi_delayed, prompt_nu_fission, delayed_nu_fission, and beta) and this comment section seems unnecessarily large as more cross sections are added.